components/team/Members.tsx

```diff
@@ -46,7 +46,10 @@ const Members = ({ team }: { team: Team }) => {
       return;
     }
 
-    const sp = new URLSearchParams({ memberId: member.userId });
+    const sp = new URLSearchParams({
+      memberId: member.userId,
+      memberRole: member.role,
+    });
 
     const response = await fetch(
       `/api/teams/${team.slug}/members?${sp.toString()}`,
```

Comment:
Passing in a role via API would mean someone can change it and get the request to pass. Instead member role should be read from DB in the API handler and then validated against that

Reply:
Ok got it implementing changes.

Reply:
> Passing in a role via API would mean someone can change it and get the request to pass. Instead member role should be read from DB in the API handler and then validated against that

@deepakprabhakara updated please check and advise